staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -433,3 +433,10 @@ func LogQemuLogLine(logger *FilteredLogger, line string) {
 		"msg", line,
 	)
 }
+func EventLoggerSink(event *v1.Event) {
+	if event.Type == v1.EventTypeWarning {
+		DefaultLogger().ObjectRef(&event.InvolvedObject).With("event", event.Type).Reason(fmt.Errorf(event.Reason)).Error(event.Message)
+	} else {
+		DefaultLogger().ObjectRef(&event.InvolvedObject).With("event", event.Type).Reason(fmt.Errorf(event.Reason)).Info(event.Message)
+	}
+}
```

Comment:
the log message says it originates from log.go line 440 for all log messages. we need to have an offset that reflects the actual line the log line originates from. Below is an example of what i mean. 

```{"component":"virt-controller","event":"Normal","kind":"VirtualMachineInstance","level":"info","msg":"Created virtual machine pod virt-launcher-vmi-ephemeral-x4ch5","name":"vmi-ephemeral","namespace":"default","pos":"log.go:440","reason":"SuccessfulCreate","timestamp":"2021-08-20T19:13:27.001810Z","uid":"5e93fea8-4c9e-452a-9daa-b8d2230b965e"}```

Reply:
@rmohr any ideas here?

Reply:
We introduced a new layer from the stack perspective. We have the depth in some helpers encoded. For instance

```
func (l FilteredLogger) msgf(msg string, args ...interface{}) {
	l.log(3, "msg", fmt.Sprintf(msg, args...))
}
```

where we have `3`. A small extension to `log.go` where one can set the offset should be good enough.